### PR TITLE
[Feature] new breadcrumb

### PR DESCRIPTION
**Pull request by rosamaggi**
- Update style with new version of `elements`
- Modify breadcrumb with new style
- refresh of the breadcrumb by calling the menu-api web-service: `http://menu-api-siblings:3001/menus/` with the option `siblings` or `breadcrumb`

### CODE REVIEW

File: wp-theme-2018/archive.php [hunk 1]
```diff
@@ -12,6 +12,7 @@
 ?>
 
 <div class="<?php echo $containerClasses ?>">
+    <?php get_sidebar(); ?>
```
Comment (ponsfrilus) [inline wp-theme-2018/archive.php:15]: Rest of file seems to have tabs instead of spaces.

File: wp-theme-2018/assets/css/elements.css [hunk 2]
```diff
@@ -102,6 +94,7 @@ p {
 
 abbr[title],
 abbr[data-original-title] {
+  text-decoration: underline;
```
Comment (ponsfrilus) [inline wp-theme-2018/assets/css/elements.css:97]: Duplicate.

File: wp-theme-2018/assets/css/elements.css [hunk 3]
```diff
@@ -3660,14 +3130,14 @@ input[type=button].btn-block {
   border-radius: 2px;
 }
 .custom-checkbox .custom-control-input:checked ~ .custom-control-label::after {
-  background-image: url("data:image/svg+xml;charset=utf8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 8 8'%3E%3Cpath fill='%23fff' d='M6.564.75l-3.59 3.612-1.538-1.55L0 4.26 2.974 7.25 8 2.193z'/%3E%3C/svg%3E");
+  background-image: url("data:image/svg+xml;charset=utf8,%3Csvg xmlns=%27http://www.w3.org/2000/svg%27 viewBox=%270 0 8 8%27%3E%3Cpath fill=%27%23fff%27 d=%27M6.564.75l-3.59 3.612-1.538-1.55L0 4.26 2.974 7.25 8 2.193z%27/%3E%3C/svg%3E");
```
Comment (ponsfrilus) [inline wp-theme-2018/assets/css/elements.css:3133]: ' → %27: who can confirm that is what we want.

File: wp-theme-2018/header.php [hunk 4]
```diff
@@ -169,7 +169,13 @@ function _get_our_soa () {
 					'walker' => new EPFL_Theme2018_Root_Menu_Walker()
 				) );
 			} else {
-				require_once(__DIR__ . "/header-top-menu-fallback.php");
+                ?>
```
Comment (ponsfrilus) [inline wp-theme-2018/header.php:172]: Indentation

File: wp-theme-2018/home.php [hunk 5]
```diff
@@ -12,13 +12,14 @@
 ?>
 
 <div class="<?php echo $containerClasses ?>">
+    <?php get_sidebar(); ?>
```
Comment (ponsfrilus) [inline wp-theme-2018/home.php:15]: Tab ?

File: wp-theme-2018/page.php [hunk 6]
```diff
@@ -12,6 +12,7 @@
 ?>
 
 <div class="<?php echo $containerClasses ?>">
+    <?php get_sidebar(); ?>
```
Comment (ponsfrilus) [inline wp-theme-2018/page.php:15]: Tab ?

File: wp-theme-2018/sidebar.php
Comment (ponsfrilus) [inline wp-theme-2018/sidebar.php:1]: Tabs vs spaces

File: wp-theme-2018/single.php [hunk 7]
```diff
@@ -17,6 +17,7 @@
 ?>
 
 	<div class="<?php echo $containerClasses ?>">
+        <?php get_sidebar(); ?>
```
Comment (ponsfrilus) [inline wp-theme-2018/single.php:20]: Tab

File: wp-theme-2018/template-parts/breadcrumb.php [hunk 8]
```diff
@@ -123,92 +135,205 @@ function get_current_item($items) {
...
-            $crumb_item = array_key_exists($index, $items)? $items[$index]: false;
-        }
-    } else {
-        // make at least the only element printed
-        $crumb_items = [$crumb_item];
+
+function render_siblings($siblings_items, $crumb_item) {
+    $siblings = '';
+    foreach ($siblings_items as $sibling) {
+        if ($crumb_item['title'] == $sibling['title'])
```
Comment (ponsfrilus) [inline wp-theme-2018/template-parts/breadcrumb.php:163]: Curly braces ?

File: wp-theme-2018/template-parts/breadcrumb.php [hunk 9]
```diff
@@ -123,92 +135,205 @@ function get_current_item($items) {
...
+        //TODO write logs somewhere ---- echo 'Failed to retrieve data from the API.';
+    } else {
+        $data = json_decode($response, true)['result'];
+        if ($data !== null) {  //With the API response we set both transients
+            /*set_transient($shortTransientName, $data, $shortCacheRefreshInterval);
+            set_transient($longTransientName, $data, $longCacheRefreshInterval);*/
+            return $data;
         } else {
-            $crumbs[] = get_rendered_crumb_item($crumb_item, False);
+            //TODO write logs somewhere ---- echo 'Failed to parse JSON response.';
```
Comment (ponsfrilus) [inline wp-theme-2018/template-parts/breadcrumb.php:249]: Maybe something with `error_log();`?

**Review by ponsfrilus — APPROVED** [review]
I'm approving it but you may want to check my comments and configure your IDE to uses real tabs when needed.

File: wp-theme-2018/template-parts/breadcrumb.php (hunk 8 above)
Comment (rosamaggi) [reply wp-theme-2018/template-parts/breadcrumb.php]: Done

**Pull request merged**